StRoot/StBFChain/StBFChain.cxx

```diff
@@ -18,6 +18,10 @@
 #include "StEnumerations.h"
 #include "TTree.h"
 #include "TEnv.h"
+
+#include <map>
+#include <string>
```

Comment:
Looks like you removed the `gSharedLibs` line that was forcing these includes, so perhaps remove these includes too?